### ISSUE TITLE
Some questions about the size of pixel feature

### ISSUE THREAD

**Issue by 3learning12340** [wentaozhu/DeepLung#117]
Thank you for sharing the code. If I remember correctly, the maximum size of the nodules  is greater than 16 in LIDC/IDRI dataset. However,  for pixel feature, you use the cropped size of 16x16x16 and center as the detected nodule center in the experiments. Therefore, I think the pixel feature may not contain all information of nodules. what's reason that you crop 16x16x16 cube voxel ? just to save GPU meomery?  I am looking  forward to your reply.

**Comment by 3learning12340** [comment]
I've duplicated most of the code and I read your paper carefully. However, I still have some doubts about the paper and I want to ask you some questions about the paper while you are free. I used to do some study on the maglignant and benign nodule classification in  two-dimensional CT images. I want to do some study on the detection and classification in the three-dimensional images and I am really thankful for your code sharing, so that I can have a better start. Can I make a friend with you by adding Wechat, QQ or facebook friends, My google email is perfect0insist@gmail.com.
